tests/hare/test_hare.py

```diff
@@ -72,7 +72,7 @@ def query_hare_output_set(indx, namespace, client_po_name):
 
 
 def test_hare_sanity(load_config, setup_clients, save_log_on_exit):
-    delay = int(testconfig['client']['args']['hare-round-duration-sec']) * 7
+    delay = int(testconfig['client']['args']['hare-round-duration-sec']) * 8
```

Comment:
why 8? whats 8? name the magic num

Reply:
This is only assessment to the delay before checking that all nodes report in ES.
@gavraz any comments on that?

Reply:
Yes. It should be 5. You can name it NUM_OF_EXPECTED_ROUNDS